libs/escrow-utils/src/constants.ts

```diff
@@ -152,6 +152,33 @@ export const NETWORK_CONFIG: { [key: number]: NetworkConfig } = {
       },
     },
   },
+  11155111: {
+    SUBGRAPH: 'smart-invoice-sepolia/v0.0.7',
```

Comment:
@sshmm could we get the subgraph id, factory addresses and resolvers from smartinvoicexyz packages ?